src/test/kotlin/org/rustSlowTests/cargo/runconfig/RunConfigurationTestBase.kt

```diff
@@ -71,13 +67,16 @@ abstract class RunConfigurationTestBase : RsWithToolchainTestBase() {
             ?: error("Can't create run configuration settings")
     }
 
-    protected fun execute(configuration: RunConfiguration): ExecutionResult {
+    protected fun execute(
+        configuration: RunConfiguration,
+        runner: ProgramRunner<*> = CargoCommandRunner()
```

Comment:
Looks like we should pass runner manually
It should be chosen by execution environment